.travis.yml

```diff
@@ -1,10 +1,11 @@
 language: cpp
 os: linux
+dist: bionic
```

Comment:
All machines are set to xenial again though. We could use 1 or 2 bionic jobs

Reply:
Yeah xenial is a bit outfashioned, which is also why we upgrade `gcc` in those jobs. This travis file has to be completely fixed anyway...